src/olympia/devhub/utils.py

```diff
@@ -307,3 +312,48 @@ def add_dynamic_theme_tag(version):
     files = version.all_files
     if any('theme' in file_.webext_permissions_list for file_ in files):
         Tag(tag_text='dynamic theme').save_tag(version.addon)
+
+
+def get_addon_akismet_reports(user, user_agent, referrer, upload=None,
+                              addon=None, data=None):
+    if not waffle.switch_is_active('akismet-spam-check'):
+        return []
+    assert addon or upload
+    properties = ('name', 'summary', 'description')
+
+    if upload:
+        addon = addon or upload.addon
+        data = data or Addon.resolve_webext_translations(
+            parse_addon(upload, addon, user), upload)
+
+    if not data:
+        return []  # bail early if no data to skip Translation lookups
+    if addon and addon.has_listed_versions():
+        translation_ids = (
+            getattr(addon, prop + '_id', -1) for prop in properties)
```

Comment:
Where does `-1` come from?

Reply:
it's just a pk that won't exist in the filter.

Reply:
This should not be send to the database tbh, so doing something like

```python

translation_ids = [getattr(addon, prop + '_id', None) for prop in properties]
# Filter empty/not existing values
translation_ids = [prop for prop in translation_ids if prop]
```

may work a bit better since that'd filter away not existing values. We don't care about empty values anyway

Reply:
*shrug* it's an extra line of calculation to filter out values from a list that will never match, but can do.

Reply:
https://github.com/mozilla/addons-server/pull/9353/commits/c1b6713c043c20456823b858bc0b3e8cf8185426